lte/cloud/configs/nprobe.yml

```diff
@@ -10,20 +10,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# operator_id represents the mobile operator identifier
-# update_interval_secs sets the priodic time between runs in seconds.
-# backoff_interval_secs sets the backoff time when remote records collector is not
-# available.
-# delivery_function_address defines the address of the remote server collecting records.
-# exporter_key provides the absolute path to exporter tls private key.
-# exporter_crt provides the absolute path to exporter tls certificate.
-# skip_verify_server enables exporter to skip server tls certificate verifications.
+# operatorID represents the mobile operator identifier
+# updateIntervalSecs sets the priodic time between runs in seconds
+# backoffIntervalSecs sets the backoff time when remote records collector is not available
+# maxExportRetries sets the number of retries when exporting a record
+# deliveryServer defines the address of the remote server collecting records
+# exporterKey provides the absolute path to exporter tls private key
+# exporterCrt provides the absolute path to exporter tls certificate
+# verifyServer enables exporter to skip server tls certificate verifications
```

Comment:
Move all these docstring comments directly above their key-value pair -- otherwise we have to hunt down-and-back to see what each comment refers to